lib/gnat.ex

```diff
@@ -11,6 +11,7 @@ defmodule Gnat do
     host: 'localhost',
     port: 4222,
     tcp_opts: [:binary],
+    timeout: 3_000,
```

Comment:
should we call this a `connect_timeout`? Looking at it again today I think the name `timeout` might confusing